lib/cli.js

```diff
@@ -124,9 +125,14 @@ exports.run = () => {
 	if (cli.flags.tap && !cli.flags.watch) {
 		reporter = tapReporter();
 	} else if (cli.flags.verbose || isCi) {
-		reporter = verboseReporter();
+		reporter = verboseReporter({
+			disableColors: cli.flags.color !== undefined
```

Comment:
@novemberborn By default, if the user doesn't specify either `--color` or `--no-color`, it's not set. 

If we add the following, it will be `true` if not specified, but then we wouldn't know when it's not specified and should be auto-detected:

```js
const cli = meow(``, {
	boolean: ['color'],
	default: {
		color: true
	}
});
```